cloudbuild.yaml

```diff
@@ -30,5 +30,5 @@ steps:
     - 'DATAPLEX_TASK_SA=${_DATAPLEX_TASK_SA}'
 timeout: 3600s
 options:
-  machineType: 'E2_HIGHCPU_8'
```

Comment:
Can we make this change in a separate/independent PR?

Reply:
I will revert the change, this was for fixing the intermittent timeout issue.